packages/coordinator/src/services/translation/response/common.ts

```diff
@@ -172,130 +162,14 @@ export function convertResourceToBundleEntry(resource: fhir.Resource): fhir.Bund
   }
 }
 
-export interface TranslatedAgentPerson {
-  practitionerRole: fhir.PractitionerRole
-  practitioner?: fhir.Practitioner
-  healthcareService?: fhir.HealthcareService
-  locations: Array<fhir.Location>
-  organization?: fhir.Organization
-}
-
 export function roleProfileIdIdentical(agentPerson1: hl7V3.AgentPerson, agentPerson2: hl7V3.AgentPerson): boolean {
   return agentPerson1.id._attributes.extension === agentPerson2.id._attributes.extension
 }
 
-export function translateAgentPerson(agentPerson: hl7V3.AgentPerson, prescriptionType?: string): TranslatedAgentPerson {
-  const representedOrganization = agentPerson.representedOrganization
-
-  if (prescriptionRefactorEnabled()) {
-    const practitionerRole = createRefactoredPractitionerRole(agentPerson)
-    const locations = createLocations(representedOrganization)
-
-    return {
-      practitionerRole,
-      locations
-    }
-  } else {
-    if (shouldHavePrimaryCareFormat(prescriptionType)) {
-      const organization = createOrganization(representedOrganization)
-      const practitioner = createPractitioner(agentPerson)
-      const practitionerRole = createPractitionerRole(agentPerson, practitioner.id)
-      practitionerRole.organization = fhir.createReference(organization.id)
-
-      const healthCareProviderLicenseOrganization = representedOrganization.healthCareProviderLicense?.Organization
-      if (healthCareProviderLicenseOrganization) {
-        organization.partOf = {
-          identifier: getOrganizationCodeIdentifier(healthCareProviderLicenseOrganization.id._attributes.extension),
-          display: healthCareProviderLicenseOrganization.name?._text
-        }
-      }
-
-      const translatedAgentPerson: TranslatedAgentPerson = {
-        practitionerRole,
-        practitioner,
-        healthcareService: null,
-        locations: [],
-        organization
-      }
-
-      return translatedAgentPerson
-    } else {
-      const healthCareOrganization = representedOrganization.healthCareProviderLicense?.Organization
-      let hl7Organization = representedOrganization
-      if (healthCareOrganization) {
-        hl7Organization = {
-          ...representedOrganization,
-          id: healthCareOrganization.id,
-          name: healthCareOrganization.name
-        }
-      }
-      const organization = createOrganization(hl7Organization)
-      const practitioner = createPractitioner(agentPerson)
-      const practitionerRole = createPractitionerRole(agentPerson, practitioner.id)
-      practitionerRole.organization = fhir.createReference(organization.id)
-      const locations = createLocations(representedOrganization)
-
-      const healthcareService = createHealthcareService(representedOrganization, locations)
```

Comment:
we should remove this function, I think this was the only place that called it.

Reply:
Yeah the only other place this appears to be used is in the organization tests